common/djangoapps/student/tests/test_login_registration_forms.py

```diff
@@ -42,10 +45,11 @@ def _third_party_login_url(backend_name, auth_entry, course_id=None, redirect_ur
 @ddt.ddt
 @override_settings(MODULESTORE=MODULESTORE_CONFIG)
 @unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
-class LoginFormTest(ModuleStoreTestCase):
+class LoginFormTest(UrlResetMixin, ModuleStoreTestCase):
     """Test rendering of the login form. """
-
+    @patch.dict(settings.FEATURES, {"ENABLE_COMBINED_LOGIN_REGISTRATION": False})
```

Comment:
The ticket tracking this work ([ECOM-978](https://openedx.atlassian.net/browse/ECOM-978)) states that updating the test suite to point to the new pages instead of the old pages is "not part of this ticket." As such, I've added the above the force the new pages off for these tests; I've done the same for the [`RegisterFormTest`](https://github.com/edx/edx-platform/pull/6815/files#diff-7710988c802f76eb8a3f8ca83361870bR162) class.